third_party/maya/lib/usdMaya/usdWriteJob.h

```diff
@@ -31,15 +31,18 @@
 #include "usdMaya/Chaser.h"
 
 #include "usdMaya/util.h"
-#include "usdMaya/MayaPrimWriter.h"
 #include "usdMaya/ModelKindWriter.h"
 
+#include "usdMaya/usdWriteJobCtx.h"
+
+#include <maya/MObjectHandle.h>
+
 #include <string>
 
 PXR_NAMESPACE_OPEN_SCOPE
 
 
-class usdWriteJob
+class usdWriteJob : public usdWriteJobCtx
```

Comment:
What's the rationale here for inheriting? It seems like this should just have a usdWriteJobCtx object instead?